shared/js/background/redirect.es6.js

```diff
@@ -124,7 +126,7 @@ function handleRequest (requestData) {
         }
 
         // count and block trackers. Skip things that matched in the trackersWhitelist unless they're first party
-        if (tracker && !(tracker.action === 'ignore' && tracker.reason !== 'first party')) {
```

Comment:
have we tested what will happen when blocking is disabled? Is this going to trigger "this page is temporary disabled" banner in the popup? Or will it look like it's blocking stuff or will it say that there are no trackers? I think we may need to understand that if we ever have to disable it.